File: w08/d03/Classwork/mechanics/ex8.js
// Create a function that takes a hash of favorite colors and uses Object.keys to print out "[name] favorite color is [color]" for each name

var people_hash = {
        "tim": "blue",
        "batman": "black",
        "flash": "red"
}

function favColors() {
  for(var value in people_hash) {
  console.log(value + " has a favorite color of " + people_hash[value]);
  }
}

favColors(people_hash)

File: w09/d02/Homework/mechanics/ex8.rb
# Create a function that takes a hash of favorite colors and uses Object.keys to print out "[name] favorite color is [color]" for each name


people_hash = {
        tim: "blue",
        batman: "black",
        flash: "red"
}

def fav_color(colors)
    colors.each do |k,v|
    puts "#{k} has a favorite color of #{v}"
  end
end

fav_color(people_hash)